cmake/LYTestWrappers.cmake

```diff
@@ -279,6 +279,9 @@ function(ly_add_test)
     set(LY_TEST_PARAMS "${LY_TEST_PARAMS}#${ly_add_test_TIMEOUT}")
     # Store the params for this test target
     set_property(GLOBAL APPEND PROPERTY LY_ALL_TESTS_${test_target}_PARAMS ${LY_TEST_PARAMS})
+
+    # Allow TIAF to disable tests of supported types from being run by CTest 
+    o3de_test_impact_apply_test_properties(${LY_ADDED_TEST_NAME} ${ly_add_test_TEST_LIBRARY})
```

Comment:
Perhaps TIAF-labels can be added, instead of disabling? This way CTest can still be directly used "normally" regardless of how TIAF marked the tests. Then conditionally running TIAF tests involves selecting unions/exclusions of labels.

Reply:
I agree with @Kadino . Having a label, allows the Ctest command line to exclude test that are marked with it via the [-LE](https://cmake.org/cmake/help/latest/manual/ctest.1.html#cmdoption-ctest-LE) label exclusion argument.